src/controls/adaptiveCardHost/fluentUI/Actions.tsx

```diff
@@ -0,0 +1,231 @@
+import { ActionButtonState, ActionIconPlacement, defaultHostConfig, ExecuteAction, OpenUrlAction, ShowCardAction, SubmitAction, ToggleVisibilityAction } from "adaptivecards";
+import { BaseButton, Button, CompoundButton, createTheme, DefaultButton, ITheme, PrimaryButton } from "office-ui-fabric-react";
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { getDefaultFluentUITheme, hostCapabilitiesFluentUIThemeKey } from "./FluentUI";
+import { createDiv } from "./Shared";
+const redFluentUIPalette = require("../themes/redPalette.json");
+
+export class FluentUIExecuteAction extends ExecuteAction {
```

Comment:
The first 5 classes (FluentUIExecuteAction, FluentUIOpenUrlAction, FluentUIShowCardAction, FluentUISubmitAction, FluentUIToggleVisibilityAction) seem to have the exact same code for `actionClickHandler` and `render` methods. 
If they will likely remain the same in the future, could this be refactored to avoid duplicated code?

Reply:
Ok, I'll check in all files and I'll change that

Reply:
Yes, I know! Do you have any suggestions on how to resolve this?

Reply:
The problem is this: Every Actions (ExecuteAction, OpenUrlAction, ShowCardAction, SubmitAction, ToggleVisibilityAction) have a base class named Action that is an abstract class... I'm not able to create a generic class that extends an abstract class... this is the reason I decided to create 5 classes whit the same code... If you have a suggestion, it will very appreciated :)

Reply:
I was just looking into it not and found the same issues 😅 didn't realise that when I added the comment. I will have another quick look but this can go as is if we can't find an elegant solution

Reply:
Nope, can't think of a better solution so leave it as is. Sorry for that